lib/particle.js

```diff
@@ -0,0 +1,84 @@
+let mcdata
+
+module.exports = loader
+
+function loader (registry) {
+  mcdata = registry
+  return Particle
+}
+
+class Particle {
+  constructor (id, longDistance, position, offset, particleData, particles, data) {
```

Comment:
Do we need this class with all the getters and setters?

Sounds like a simple Particle.fromNetwork(packet) => Particle would suffice. Using constructors like this is not future proof if fields change.

Reply:
Changed to a function